its/src/test/java/com/sonar/it/csharp/NoSonarTest.java

```diff
@@ -40,19 +40,16 @@
   @BeforeClass
   public static void init() throws Exception {
     TestUtils.reset(ORCHESTRATOR);
-
-    // Without setting the testProjectPattern, the NoSonarTest project is considered as a Test project :)
-    Tests.analyzeProject(temp, PROJECT, "class_name", "sonar.msbuild.testProjectPattern", "noTests");
+    Tests.analyzeProject(temp, PROJECT, "class_name");
```

Comment:
Replaced deprecated `sonar.msbuild.testProjectPattern` with `<SonarQubeTestProject>false</SonarQubeTestProject>` to simplify the situation (new scanner is not released yet).